Добавлены новые дочерние  страницы. Подключили firebase для хранения …

sofka opened the pull request:
…объекта массива покемонов. Реализовано добавление и переворот карточки с запоминанием в бд.

Через contextapi реализована передача выбранных покемонов с одной страницы на другую.
Изменена карточка покемона, некоторые стили прокидываются сверху.

Очень долго провозилась со стилями, сначала карточки долго не отображались, через чат разобралась.
Так как карточки не отображались, вообще не могла понять, что и куда нужно сделать, как будо описания не хватило, тоже в чате уточняла.
 Потом вылезла ошибка Cannot update a component (`GamePage`) while rendering a different component (`StartPage`). Сидела гуглила, не смогла понять, как её решить. В чате написал, что на следующем уроке расскажешь, как с этим бороться.
Ну и следующая ошибка из-за которой отложилась сдача дз. Сначала переделала firebaseconfig и useeffect как во второй части видео к дз, потом переход на Board выдал ошибку Firebase App named '[DEFAULT]' already exists. Я её исправила, как в чате посоветовали, плюс и гугл такой ответ выдаёт, но почему происходит обрыв соединения с бд я не поняла. И как корректно решить, тоже.

sofka committed:
Добавлены новые дочерние  страницы. Подключили firebase для хранения объекта массива покемонов. Реализовано добавление и переворот карточки с запоминанием в бд.
Через contextapi реализована передача выбранных покемонов с одной страницы на другую.
Изменена карточка покемона, некоторые стили прокидываются сверху.

Очень долго провозилась со стилями, сначала карточки долго не отображались, через чат разобралась.
Так как карточки не отображались, вообще не могла понять, что и куда нужно сделать, как будо описания не хватило, тоже в чате уточняла.
 Потом вылезла ошибка Cannot update a component (`GamePage`) while rendering a different component (`StartPage`). Сидела гуглила, не смогла понять, как её решить. В чате написал, что на следующем уроке расскажешь, как с этим бороться.
Ну и следующая ошибка из-за которой отложилась сдача дз. Сначала переделала firebaseconfig и useeffect как во второй части видео к дз, потом переход на Board выдал ошибку Firebase App named '[DEFAULT]' already exists. Я её исправила, как в чате посоветовали, плюс и гугл такой ответ выдаёт, но почему происходит обрыв соединения с бд я не поняла. И как корректно решить, тоже.

sofka commented:
Домашка оказалась сложной. Чёт прям много времени потратила, какие-то ошибки вылетали вечно. Да и со стилями я бы разобралась быстрее, если бы сообразила про className  хитрый.

ZarMarathon reviewed (approved):
бывает) но по дз всё огонь в итоге, аппрув!

sofka committed:
Переделала дз по видео.
Получается что добавила ограничение,что выделить можно только 5 покемонов.
Добавила хук useLocation
